packages/syntax-core/src/MiniActionCard/MiniActionCard.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { StoryObj, Meta } from "@storybook/react";
+import MiniActionCard from "./MiniActionCard";
+import image from "../../../../apps/storybook/assets/images/book.svg";
+
+export default {
+  title: "MiniActionCard",
+  component: MiniActionCard,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1206-4420&t=yFh7Ijhf6PU7Lin3-0",
+    },
+  },
+  tags: ["autodocs"],
+} as Meta<typeof MiniActionCard>;
+
+export const Default: StoryObj<typeof MiniActionCard> = {
+  args: {
+    children: (
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+          height: "100%",
+          gap: "24px",
+        }}
+      >
+        {/* eslint-disable-next-line @typescript-eslint/no-unsafe-assignment */}
```

Comment:
I can't figure out what's wrong with this line or why eslint was throwing this error

Reply:
Also not sure what's wrong with Vercel. I don't have access to it

Reply:
@jliotta Did you not see the following warning in your editor? 

> Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-elementeslint[@next/next/no-img-element](https://nextjs.org/docs/messages/no-img-element)

![image](https://user-images.githubusercontent.com/127199/226396585-39226c41-4c7a-4eff-8808-492c65e35b82.png)

Issue here is that we have the `next` ESLint rules enable for all or `syntax-core`. That might make sense for most of our components but not our stories. So I disabled that.

Reply:
@jliotta As for vercel access, I just invited you

The error was the following:
> @cambly/syntax-core:build: src/index.tsx(4,28): error TS2307: Cannot find module './MiniActionCard' or its corresponding type declarations.
17:25:21.579

And the issue was a wrong import:
![image](https://user-images.githubusercontent.com/127199/226400284-df604fc3-35b7-4be3-891d-575fbb73a24d.png)

Changing that to `MiniCard/MiniCard` fixes the issue:
![image](https://user-images.githubusercontent.com/127199/226400427-0c2df77e-3a89-4175-8d06-56489a040fb0.png)


I also added `pnpm run build` to GitHub CI so folks will be able to see these errors there instead:
![Screenshot 2023-03-20 at 9 10 00 AM](https://user-images.githubusercontent.com/127199/226401885-d32412fe-2ead-44f4-a7eb-92e0f9e0ca5d.png)